lib/mayaUsd/ufe/UsdContextOps.cpp

```diff
@@ -196,34 +199,6 @@ struct WaitCursor
 };
 
 #ifdef UFE_V3_FEATURES_AVAILABLE
-//! \brief Create a Prim and select it:
-class UsdUndoAddNewPrimAndSelectCommand : public Ufe::CompositeUndoableCommand
-{
-public:
-    UsdUndoAddNewPrimAndSelectCommand(const UsdUfe::UsdUndoAddNewPrimCommand::Ptr& creationCmd)
-        : Ufe::CompositeUndoableCommand({ creationCmd })
-    {
-    }
-
-    void execute() override
-    {
-        auto addPrimCmd
-            = std::dynamic_pointer_cast<UsdUfe::UsdUndoAddNewPrimCommand>(cmdsList().front());
-        addPrimCmd->execute();
-        // Create the selection command only if the creation succeeded:
-        if (!addPrimCmd->newUfePath().empty()) {
-            Ufe::Selection newSelection;
-            newSelection.append(Ufe::Hierarchy::createItem(addPrimCmd->newUfePath()));
-            append(Ufe::SelectionReplaceWith::createAndExecute(
-                Ufe::GlobalSelection::get(), newSelection));
-        }
-    }
-
-#ifdef UFE_V4_FEATURES_AVAILABLE
-    std::string commandString() const override { return cmdsList().front()->commandString(); }
-#endif
-};
-
 //! \brief Create a working Material and select it:
 class InsertChildAndSelectCommand : public Ufe::CompositeUndoableCommand
```

Comment:
Any reason this command wasn't moved to a separate file?

Reply:
It seemed a very specific combination of two commands for the context menu. I could move it.

Reply:
The explanation makes sense. If its not reusable then leave it in ContextOps